db/models/articles-model.js

```diff
@@ -1,5 +1,14 @@
 const db = require("../connection");
 
+const readArticles = () => {
+  return db.query(`SELECT * FROM articles ORDER BY created_at DESC;`).then(({ rows }) => {
```

Comment:
As per my previous comment, please amend this SQL query so it joins the comments table and creates the comment_count. 🙂